exercises/practice/matrix/zcl_matrix.clas.testclasses.abap

```diff
@@ -0,0 +1,106 @@
+CLASS ltcl_matrix DEFINITION FOR TESTING RISK LEVEL HARMLESS DURATION SHORT FINAL.
+
+  PRIVATE SECTION.
+    TYPES integertab TYPE STANDARD TABLE OF i WITH EMPTY KEY.
```

Comment:
suggest reusing `zcl_matrix=>integertab` instead of having the type definition multiple times

Reply:
good idea, fixed